### PR TITLE
fix: clear existing paths when creating paths

### PR DESCRIPTION
**Pull request by dodumosu**
this commit works around NetworkX's failure (by design?)
to generate a proper shortest path when edges are unordered.

see: #334

### CODE REVIEW

File: apollo/locations/utils.py [hunk 1]
```diff
@@ -46,6 +46,9 @@ def import_graph(graph, location_set, fresh_import=False):
                 flag_modified(location_type, 'name_translations')
                 location_type.save()
 
+            else:
+                return
```
Comment (takinbo) [inline apollo/locations/utils.py]: is there a reason why the return statement has no value?

**Comment by dodumosu** [comment]
The function is only used for its side effect of building the location
levels closure table. The early return skips trying to build the closures
if any of the submitted data doesn't exist in the database and is not new
data.

**Comment by takinbo** [comment]
@dodumosu this issue is still outstanding. Here are the steps to reproduce the error.

Create a regular set of admin divisions and then edit it to add a division in between the two, you get something that looks like this:

screenshot-1560255694

**Comment by takinbo** [comment]
@dodumosu issue is still not resolved after the last update. See the screenshots below.

Before save
screenshot-1560319926

After save
screenshot-1560319937

**Pull request merged**